test/integration/generated/appconfiguration/package.json

```diff
@@ -81,5 +81,10 @@
     "docs": "echo skipped"
   },
   "sideEffects": false,
+  "//metadata": {
+    "constantPaths": [
+      { "path": "src/AppConfigurationClient.ts", "prefix": "packageDetails" }
```

Comment:
The code changes look good. But, I have a question. Why is it formatted in one file and not formatted to two lines in other files?

Reply:
It might be caused by VScode. I'm actually not sure, all of these files are generated so I didn't touch them.
Going to take a look locally to see if this was caused by my dev environment.

Reply:
After some tests and research, I found that this is done by Prettier, it will try to put all elements in a single line only if they fit with the default print width, as it written [here](https://prettier.io/docs/en/rationale.html#multi-line-objects). So, paths being larger than others is causing this discrepancy between the generated files.